src/components/TwoRowsButton/link.tsx

```diff
@@ -12,7 +12,7 @@ type ITwoRowsButtonLinkProps = {
   children?: React.ReactNode
   active?: boolean
   href?: string
-} & React.ButtonHTMLAttributes<HTMLAnchorElement>
+} & React.HTMLProps<HTMLAnchorElement>
```

Comment:
Had to adjust the prop type so I could add the `target` attribute.

Reply:
I think we could have used the `Link` Component and extended from there. It's fine as it is for now to move forward.

Reply:
Good idea! Forgot we had that Link component. Went ahead and added it in since it was a quick change.

Reply:
Thanks for addressing this @julieg18.